README.md

```diff
@@ -1,28 +1,25 @@
-# `kcp` is a minimal Kubernetes API server
+# `kcp` is a control plane for applications running across multiple Kubernetes clusters
 
 ![build status badge](https://github.com/kcp-dev/kcp/actions/workflows/ci.yaml/badge.svg)
 
-How minimal exactly? `kcp` doesn't know about [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)s or [`Node`](https://kubernetes.io/docs/concepts/architecture/nodes/)s, let alone [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s, [`Service`](https://kubernetes.io/docs/concepts/services-networking/service/)s, [`LoadBalancer`](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/)s, etc. when started.
-
-By default, `kcp` only knows about:
+`kcp` is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver. By default, `kcp` only knows about:
```

Comment:
I would talk about logical clusters right up front here.

BTW, to me, the term "logical cluster" is a bit of a misdirection because it focuses on the "cluster", which primarily brings compute to mind.  If I understand correctly, however, the "logical cluster" concept is about virtualizing the api service.  Or, a little more concretely and clearly, introducing a higher level of division/grouping above namespace.  Perhaps terms like "logical api service" or "virtual api service" or "higher level namespace" would be more on point.

Reply:
Updated